pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -28,29 +29,56 @@ import (
 	statsv1alpha1 "antrea.io/antrea/pkg/apis/stats/v1alpha1"
 )
 
+var (
+	fakeClock = testing2.NewFakeClock(time.Now())
+)
+
 func TestREST(t *testing.T) {
-	r := NewREST()
+	r := NewREST(fakeClock)
 	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.New())
 	assert.Equal(t, &statsv1alpha1.NodeLatencyStats{}, r.NewList())
 	assert.False(t, r.NamespaceScoped())
 }
 
 func TestRESTCreate(t *testing.T) {
-	summary := &statsv1alpha1.NodeLatencyStats{
-		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
-		PeerNodeLatencyStats: nil,
-	}
-	expectedObj := &statsv1alpha1.NodeLatencyStats{
-		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
-		PeerNodeLatencyStats: nil,
+	tests := []struct {
+		name        string
+		summary     *statsv1alpha1.NodeLatencyStats
+		expectedObj *statsv1alpha1.NodeLatencyStats
+	}{
+		{
+			name: "create with time added",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1", CreationTimestamp: metav1.Time{Time: fakeClock.Now()}},
+				PeerNodeLatencyStats: nil,
+			},
+			expectedObj: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1", CreationTimestamp: metav1.Time{Time: fakeClock.Now()}},
+				PeerNodeLatencyStats: nil,
+			},
```

Comment:
this test should validate that the timestamp is not modified by `Create`. You cannot really validate that unless you advance the clock.